### DISCUSSION TITLE
Batch video find/play mode

### DISCUSSION THREAD

**Discussion by mrx23dot** [FreeTubeApp/FreeTube#4525] [General]
So I have a list of trailers to checkout every week, like 

```
The Curse trailer
A Murder at the End of the World trailer
Beacon 23 trailer
```

it would be great to just insert them into a multiline textbox, the app would find first video for each and put it into a playlist with autoplay.
Without searching one by one. First result is good enough.

I'm sure this would be useful for others too, like market research or whatnot.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Queueing videos will be included in the next few releases when upcoming local playlists has matured. But i dont see something like this making it into FT.  

1. A video title isnt enough because there are always multiple results. 
2. This would mean you need to include the channel name and the correct title of the video (capital sensitive) to get the proper result
3. I also dont see why this is needed because you can just create a profile and subscribe to the channels that upload those trailers

This just seems to be an unnecessary workaround for stuff that has yet to be implemented. 

If you want something like this it is best to make a custom build yourself.
